frame/bridge/ethereum/issuing/src/mock_header.rs

```diff
@@ -0,0 +1,13 @@
+
+// https://ropsten.shadow.darwinia.network/ethereum/receipt/0x6f3c3a80f22584efecdc4f5c835a092484e56b1eb05ae01c364071a870f84f02/10254220
+pub const SEND_RECEIPT_PROOF: &str = "0xf90561f9055eb893f891a0e810001f1afee20cf78babf2f8bc013de261bf91e0c33c8aec86e6c41e794ac7a0ed941a4dc5320ad5b5bc67b17e0b6f2a809d5f0e95fc8e047f1657ca4c3e3843a09e7170caf2f5d8b531c546c48fa1459f07bb770521ad0ae4611d6a1f5c6188b98080808080a0e58215be848c1293dd381210359d84485553000a82b67410406d183b42adbbdd8080808080808080b90214f90211a02e87be85c088dcd18fba2b2502d5bc15325c7f9d98c61606748e20f81ab1f17ba00900f2fe4524e02164ac1ff7e20bd257b2e8cc3b12f244ed1e14308eae721ed8a039b260b06a5b752620ec329799d29af53cd402344700919e4fdf175c730cc902a0c12afef0322982ebebe0784c445e6ac050ba968bdace65b582f7dc263204c42ea05f9e5b540267aad71dbc444cbc48ecc3b602e647ea4eaf254ab3e47dbc5b3b69a0d42c2b8d78015446e97a74c79e787f56acd31a61ea76259ae9659a9b48a2a0e6a021df06f6e3a835099a607147b2832b96459ca9efac2ac66286fe0b55aff452eaa0ba7a484171aa9a58cfe991f28409a3beafaab986048da385ebaff166a5041bb3a022fe851f7c8a9b7c81b9212504adbea63265ad4fb3e3e7554086fb5ae7d146b0a069ba47e4e735a26de3f53bc56dac562d621cd871af02afe0eff94e435d3f3c87a0777586877808e6a4c7b1cb87e7b0d0d6988b50b5429560ef1b36fc2de4b2504da0ce07dd72be37ec51381b05834cd1b38feeab308c9f84f7cb1cf394815466457fa076716c5f477cd869b818f8e3f5a847275b36f8ac84dec0100a301b23f00c6cdda07b01d139025affc2710720af13e4707e72e4f38f1a4dc36ca6b13b23b56f1aeaa02e530c12101aa796f4db3bb4a4e0be11c1bbb8c610906d843e28c183d7021b9aa0f75a1907455318d7d8813df1d2df06bcb41ea81c230a2b5766817493987c41cc80b902aff902ac20b902a8f902a5018376295cb9010000000000000000000020020000000000000000000000000000000000000000000000000000000000084000000000400000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000042000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000200000000000080000000000000000000000000000000008000000000000000000800000000800000000000000000002000000000000000000000000004000000000000000000000000000000000200000100000000000000000000000000000000000000000000000000000f9019af89b94f2d4cafe299a997127915d2e44afbc86ed7082a2f863a0ddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3efa0000000000000000000000000f2d4cafe299a997127915d2e44afbc86ed7082a2a0000000000000000000000000b2bea2358d817dae01b0fd0dc3aecb25910e65aaa00000000000000000000000000000000000000000000000000000000000000064f8fb94b2bea2358d817dae01b0fd0dc3aecb25910e65aaf842a0c2088d2d8c703cc9316b03c6aeba47f60b71c0a979c6c8ee5d60c9143511dfa5a0000000000000000000000000f2d4cafe299a997127915d2e44afbc86ed7082a2b8a0000000000000000000000000f2d4cafe299a997127915d2e44afbc86ed7082a2000000000000000000000000c1332ecce3a065bfe8babd8d28ee0b1a7c76cbcc0000000000000000000000000000000000000000000000000000000000000064000000000000000000000000d2c7008400f54aa70af01cf8c747a4473246593e0000000000000000000000000000000000000000000000000000000000000000";
```

Comment:
How about putting these mock data to the `benchmarking.rs`, since the things are only used in benchmark now.

Reply:
It is better to put these mock data in a single file. 
The data is too large.
It will cause large memory usage and response slowly for the editor.